cli/lib/creator.js

```diff
@@ -339,6 +339,10 @@ Creator.prototype.configOptionPlatforms = function configOptionPlatforms(order)
 			});
 		}
 
+		if (goodValues.mobileweb) {
+			logger.warn(__('MobileWeb platform has been deprecated.'));
```

Comment:
Can you mention 5.4.0 and 7.0.0? :-) Like: "MobileWeb platform has been deprecated in 5.4.0 and will be removed in 7.0.0".